.changeset/silly-shoes-agree.md

```diff
@@ -0,0 +1,5 @@
+---
+"@blitzjs/generator": patch
+---
+
+Remove `-cookie-prefix` appended to the cookiePrefix string
```

Comment:
I suggest making it more clear that this also fixes auth and CSRF issues for people upgrading